legend-depot-artifacts-purge/src/main/java/org/finos/legend/depot/store/artifacts/purge/services/ArtifactsPurgeServiceImpl.java

```diff
@@ -121,7 +122,9 @@ public void evict(String groupId, String artifactId, String versionId)
                 }
             });
             StoreProjectVersionData projectData = getProjectVersion(groupId, artifactId, versionId);
+            LOGGER.info(String.format("%s-%s-%s entities and generations deleted", groupId, artifactId, versionId));
```

Comment:
same as above, fo now we have entites an generations which are artifacts so keep it open and simple